pycbc/transforms.py

```diff
@@ -28,10 +28,12 @@
 from pycbc.waveform import parameters
 from pycbc.boundaries import Bounds
 from pycbc import VARARGS_DELIM
+from pycbc.waveform.waveform_modes import jframe_to_l0frame
```

Comment:
This might cause circular imports. It's usually a bad idea to import functions from sub modules in top-level modules. I originally put this function in `waveform_modes` because that's where it was being used, but it would be useful at a higher level. Maybe move it to `pycbc/pnutils.py`? Then import from there (also make sure to add an import to `pycbc/waveform/waveform_modes.py` too).

Reply:
Thanks for the help, I'll look into this asap

Reply:
I would agree with Collin that if this function becomes useful outside of `waveform_modes` then it belongs somewhere else. Collin's moved this stuff about most recently, so his suggestion seems sensible.